index.js

```diff
@@ -1,19 +1,8 @@
-var web3 = require('./lib/web3');
-var namereg = require('./lib/web3/namereg');
-
-web3.providers.HttpProvider = require('./lib/web3/httpprovider');
-web3.providers.IpcProvider = require('./lib/web3/ipcprovider');
-
-web3.eth.contract = require('./lib/web3/contract');
-web3.eth.namereg = namereg.namereg;
-web3.eth.ibanNamereg = namereg.ibanNamereg;
-web3.eth.sendIBANTransaction = require('./lib/web3/transfer');
-web3.eth.iban = require('./lib/web3/iban');
+var Web3 = require('./lib/web3');
 
 // dont override global variable
 if (typeof window !== 'undefined' && typeof window.web3 === 'undefined') {
```

Comment:
`window.web3` is still lowercase.

And shouldn't we expose already a `web3` to avoid backwards incompatibility?